gramineproject/gramine
# [RA-TLS] Adapt for the new encrypted files API

## Description of the changes 

This adapts the secret provisioning library and the example that uses it to make use of the new encrypted files APIs.

## How to test this PR? 

The `ra-tls-secret-prov` example, which is run by CI, should almost fully suffice, though the warnings and errors printed in some conditions need to be checked manually.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/587)
